backend/tournesol/lib/public_dataset.py

```diff
@@ -288,7 +299,7 @@ def write_individual_criteria_scores_file(poll_name: str, write_target) -> None:
             "public_username": criteria_score.username,
             "video": criteria_score.uid.split(UID_DELIMITER)[1],
             "criteria": criteria_score.criteria,
-            "score": criteria_score.score,
+            "score": round(criteria_score.score, FLOAT_PRECISION),
             "voting_right": criteria_score.voting_right,
```

Comment:
The individual voting right could be rounded too. But the smallest value is currently ~ 0.0184, and even smaller values could be reached in the future, so I think we should keep at least 3 digits here.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1377/commits/11e601b8d432bde585ace16d69d3cc1e4f415927